hathor/builder/sysctl_builder.py

```diff
@@ -12,14 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Optional
+
 from hathor.builder import BuildArtifacts
 from hathor.sysctl import ConnectionsManagerSysctl, Sysctl, WebsocketManagerSysctl
 
 
 class SysctlBuilder:
     """Builder for the sysctl tree."""
+
     def __init__(self, artifacts: BuildArtifacts) -> None:
         self.artifacts = artifacts
+        self.sysctl_init_file: Optional[str] = None
```

Comment:
It seems this attribute is not needed anymore.

Reply:
removed a9e338497c6c5aed78b32589334fd14fe1fde2ff

Reply:
Thanks!